services/vod-to-live/service/mosaic-hosting-deployment-manifest.yaml

```diff
@@ -0,0 +1,86 @@
+# This YAML contains the Deployment Manifest for the VOD-to-Live Service.
+# It is required when deploying the service via the Mosaic Hosting Service.
+# Refer to the README.md file for instructions on how to use this.
+
+version: '1.0'
+serviceId: 'vod-to-live-service'
+
+regularVariables:
+  # Service General Variables
+  SERVICE_ID: 'vod-to-live-service'
+  NODE_ENV: 'production'
+  LOG_LEVEL: 'DEBUG'
+  GRAPHQL_GUI_ENABLED: true
+  PORT: '${__ax_hosted__.port.api}'
+
+  # Mosaic Environment Info
+  TENANT_ID: '${__ax_hosted__.env.tenant_id}'
+  ENVIRONMENT_ID: '${__ax_hosted__.env.environment_id}'
+
+  # RabbitMQ Variables
+  RABBITMQ_PROTOCOL: '${__ax_hosted__.rmq.protocol}'
+  RABBITMQ_HOST: '${__ax_hosted__.rmq.host}'
+  RABBITMQ_PORT: '${__ax_hosted__.rmq.port}'
+  RABBITMQ_MGMT_PROTOCOL: '${__ax_hosted__.rmq.mgmt_protocol}'
+  RABBITMQ_MGMT_HOST: '${__ax_hosted__.rmq.mgmt_host}'
+  RABBITMQ_MGMT_PORT: '${__ax_hosted__.rmq.mgmt_port}'
+  RABBITMQ_VHOST: '${__ax_hosted__.rmq.vhost}'
+  RABBITMQ_USER: '${__ax_hosted__.rmq.username}'
+  RABBITMQ_PASSWORD: '${__ax_hosted__.rmq.password}'
+
+  # Managed Service URLs
+  ID_SERVICE_AUTH_BASE_URL: '${__ax_hosted__.svc.ax-id-service.auth_base_url}'
+
+  VIRTUAL_CHANNEL_MANAGEMENT_API_BASE_URL: 'REPLACE_WITH_USP_VIRTUAL_CHANNEL_MANAGEMENT_API_BASE_URL'
```

Comment:
is this going to point to the custom channel-service deployment ?

if yes, we have a similar reference from ent-service -> catalog-service, and there's a placeholder value we could use for the DNS value of another custom service

Reply:
(or maybe it refers to some USP URL - then pls ignore)

Reply:
The second one - this is the Unified Streaming Virtual Channels setup which consists of multiple micro-services. The one here is the URL to the Unified Streaming management API.